packages/core/src/hooks/useAfterFirstRender/index.ts

```diff
@@ -1,11 +1,14 @@
 import { RefObject, useEffect, useRef } from "react";
 
 export default function useAfterFirstRender(): RefObject<boolean> {
-  const isAfterFirstRender = useRef(true);
+  const isAfterFirstRender = useRef(false);
+
   useEffect(() => {
-    window.requestAnimationFrame(() => {
-      isAfterFirstRender.current = false;
-    });
+    const timer = setTimeout(() => {
```

Comment:
Why replace the RAF?

Reply:
I saw it is considered a better practice for this kind of cases
I'll give GPT the stage:

> Choose `setTimeout` with a delay of `0` over `requestAnimationFrame` - because it guarantees that the state update occurs after all rendering tasks are complete - regardless of the browser's frame rate or tab visibility, ensuring consistent execution timing across all scenarios

Reply:
Ok but will it ensure that it will return true after React's first render?

Reply:
I believe so. because when using `setTimeout` it would run the state update to run after all current tasks (including React's initial render?). 
I believe it ensures that the update occurs after React has completed its first render cycle by placing it at the end of the stack.
I also believe that this is the reason they initially went with a ref and not a state, to not force a re-render right after it.

I'm a believer